k8s-daemonset/k8s/world-rc.yml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: world
+spec:
+  replicas: 3
+  selector:
+    app: world
+  template:
+    metadata:
+      labels:
+        app: world
+    spec:
+      dnsPolicy: ClusterFirst
+      containers:
+      - name: service
+        image: <helloworld docker image>
+        imagePullPolicy: Always
+        env:
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: POD_IP
+          valueFrom:
+            fieldRef:
+              fieldPath: status.podIP
+        - name: NS
+          value: helloworld
```

Comment:
Could you use `metadata.namespace` for this, similar to how you've defined POD_NAME above?  Overall if we could avoid hardcoding the namespace altogether this would be much more portable, but I don't know how to handle dtabs, which require the namespace.